apps/ethos/repl-docmosis-service/repl-docmosis-service.yaml

```diff
@@ -14,7 +14,7 @@ spec:
       cpuRequests: "250m"
       readinessDelay: 45
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/ethos/repl-docmosis-backend:prod-77c78c6-20240709061835 #{"$imagepolicy": "flux-system:repl-docmosis-service"}
+      image: hmctspublic.azurecr.io/ethos/repl-docmosis-backend:prod-7cf0940-20240710061427 #{"$imagepolicy": "flux-system:repl-docmosis-service"}
```

Comment:
I wouldn't make this change as it will affect prod as well

Reply:
I see. I revert that one then. i was meant to keep the configuration consistent. Thanks